package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] [OPTIONAL] i would suggest installing `webpack-dev-server` and adding` "dev": "webpack serve --open",` inside the scripts object so you can run your app in the live server without having to build it on every change 

1. npm i webpack-dev-server
2.  update the scripts section in package.json to be :

```JS
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "dev": "webpack serve --open"
  },
```
3. run `npm run dev`